frontend/src/page/GamePage.tsx

```diff
@@ -104,73 +104,67 @@ export default function GamePage() {
         }
     }, [kobolds])
 
+    function lostgame() {
+        toast("You are Done!")
+        axios.delete("/api/character/lost/" + character.id)
```

Comment:
i think there is a more elegant way of doing these calls. maybe ask chatgpt for ideas. ;)